src/components/select/select.component.ts

```diff
@@ -219,7 +220,10 @@ export class SelectComponent<T> implements OnInit, OnChanges, OnDestroy, Control
             filter(() => this.allowNull),
             filter(value => !this.multiple && value !== this.getDisplay(this.value)),
             takeUntil(this._onDestroy)
-        ).subscribe(() => this.value = null);
+        ).subscribe(() => {
+            this.value = null

```

Comment:
Missing semi-colon and need to remove all the console.logs

Reply:
my bad, will try to remember to check this in future